cmd/combine/README.md

```diff
@@ -0,0 +1,65 @@
+# combine manifest lists
+
+This tool combines two Docker manifest lists ("multi-arch images") into one that provides all the platforms supported by both manifest lists.
+
+It fails if both images provide the same platforms, or if either isn't a manifest list.
+
+# demo
+
+```
+$ go run ./ \
```

Comment:
Naive question: would we want to be able to support more than 2 images in the future ?

Reply:
Maybe, if we find another architecture or OS we'd like to support that aren't covered by distroless+nanoserver.

Windows platforms are weird too, so we might end up wanting to combine distroless+windows-vX+windows-vY, etc.

I don't think we should support that for now though since for now we only need to combine two images. It should be relatively straightforward to modify this to combine multiple manifests if/when we decide to.